parallel.qmd

```diff
@@ -5,16 +5,20 @@
 source("common.R")
 ```
 
-Both `dust2` and `monty` have built-in support that makes running in parallel relatively simple when you have multiple cores available. There are two main ways to parallelise:
+Both `dust2` and `monty` have built-in support for parallelisation, making it straightforward to take advantage of having multiple cores available. There are three main ways to parallelise, depending on what part of the computation you want to speed up:
 
-1. Using workers to parallelise between chains. This is useful to anyone running multiple chains for a sampler with `monty` irrespective of whether your model is deterministic or stochastic, written in `odin2` or not.
-2. Using threads to parallelise between particles in a filter. is useful to anyone who has a stochastic model written in `odin2` and is using the filter to estimate the marginal likelihood. Using threads cans can be done in combination with using workers.
+1. **Using workers to parallelise between chains**
+   This runs each MCMC chain in its own process. It is useful whenever you are running multiple chains, regardless of whether your model is deterministic or stochastic, or whether it is written in `odin2` or not.
 
-The parallelisation is implemented in such a way that ensures that results will be identical to running in serial.
+2. **Using threads to parallelise between particles in a filter**
+   This runs different set of particles from a particle filter in parallel, within a single process. It is useful when you have a stochastic model written in `odin2` and are using the filter to estimate the marginal likelihood. Using threads can be combined with using workers.
 
-There is a third way to parallelise via manual scheduling of chains, which is of use if you want to distribute chains over, for example, the nodes of an HPC system.
+The parallelisation in both cases is implemented so that results will be identical to running in serial.
```

Comment:
Move this below point 3, and then change it to the "parallelisation of all three methods"?